src/JavascriptTestTrait.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Drupal\commerce;
```

Comment:
@smmccabe on another note, I wanted to make this a core patch. Do you want the honors?